src/CoreWCF.Primitives/src/CoreWCF/Telemetry/TelemetryContextMessageProperty.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+
+namespace CoreWCF.Telemetry;
+
+internal sealed class TelemetryContextMessageProperty
+{
+    public const string Name = "OpenTelemetry.Instrumentation.Wcf.Implementation.TelemetryContextMessageProperty";
```

Comment:
The only important thing with the name is to avoid potential collisions with some other usage. So for example, a name such as "Telemetry" would be bad as multiple libraries could end up using that. The convention I try to stick to is using the fully qualified name of the type as any other code has no legitimate reason to do that. So in this case, it would be `CoreWCF.Telemetry.TelemetryContextMessageProperty`. It's not used for anything that's sent over the wire, it's all local to the running process so in the end, it really doesn't matter, as long as you avoid potential collision/clashes with app code and what that would be doing.